app/helpers/test_executions_results_helper.rb

```diff
@@ -4,7 +4,7 @@ module TestExecutionsResultsHelper
   def get_select_history_message(execution, is_most_recent)
     msg = ''
     msg << 'Most Recent - ' if is_most_recent
-    msg << execution.created_at.in_time_zone.strftime('%B %e, %Y %l:%M%P')
+    msg << local_time(execution.created_at)
```

Comment:
Yeah, interestingly it works when you run the app itself but I cannot figure out how to get it to be included from the test suite.